src/convertToObject.js

```diff
@@ -6,7 +6,27 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const newSource = sourceString
+    .split(';')
+    .map((sourceString) => sourceString.trim())
+    .filter((sourceString) => sourceString.length > 0);
+
+  const newLine = newSource.map((line) => line.split(':'));
+
+  newLine.sort();
```

Comment:
Sorting `newLine` is unnecessary and might lead to unexpected behavior if the order of entries is important. Consider removing this line unless sorting is explicitly required by the task.